### ISSUE TITLE
Form.tsx has text that fails contrast checkers for accessibility

### ISSUE THREAD

**Issue by sehyunidaaa** [opengovsg/isomer#1766]
**Describe the bug**
Text on `Form.tsx` has text that fails contrast checkers

**Additional context**
`/packages/components/src/templates/next/components/complex/FormSG/FormSG.tsx` uses `opacity-90` and `opacity-50` on text. I think we can remove them.
